src/Silverback.Integration/Messaging/Configuration/WithCallbacksBrokerOptionsBuilderExtensions.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using Microsoft.Extensions.DependencyInjection;
+using Silverback.Util;
+
+namespace Silverback.Messaging.Configuration
+{
+    /// <summary>
+    ///     Adds methods to register <see cref="IBrokerEventsHandler"/>s to the <see cref="IBrokerOptionsBuilder"/>.
+    /// </summary>
+    public static class WithCallbacksBrokerOptionsBuilderExtensions
+    {
+        /// <summary>
+        ///     Register a <see cref="IBrokerEventsHandler"/> as singleton.
```

Comment:
The method summaries are conjugated for third person.
Anyway copy the summary style from similar extensions methods (e.g. `AddSingletonSubscriber`).

```csharp
///     Adds a singleton subscriber of the type specified in <paramref name="subscriberType" /> to the
///     <see cref="IServiceCollection" />.
```

Reply:
Thanks for pointing that out. I will fix it shortly.